src/bin/initdb/initdb.c

```diff
@@ -2655,6 +2655,7 @@ usage(const char *progname)
 	printf(_("\nLess commonly used options:\n"));
 	printf(_("  -d, --debug               generate lots of debugging output\n"));
 	printf(_("  -k, --data-checksums      use data page checksums\n"));
+	printf(_("  --preload_libraries       load libraries, same as shared_preload_libraries in postgresql.conf\n"));
```

Comment:
Let's keep the argument's name consistent with other arguments. `--preload-libraries`.